BuildAndTest.cmd

```diff
@@ -96,7 +96,7 @@ Exit /B %ERRORLEVEL%
 :CreatePublishPackage
 set Framework=%~1
 set RuntimeArg=%~2
-dotnet publish %~dp0src\BinSkim.Driver\BinSkim.Driver.csproj --no-restore -c %Configuration% -f %Framework% --runtime %RuntimeArg%
+dotnet publish %~dp0src\BinSkim.Driver\BinSkim.Driver.csproj --no-restore -c %Configuration% -f %Framework% --runtime %RuntimeArg% --self-contained
```

Comment:
>--self-contained [](http://example.com/codeflow?start=131&length=16)

fixing the warning btw,
asking us to specify if want self-contained or not. Default is self-contained, so no actual change.